generation/WinSDK/enums.json

```diff
@@ -30974,5 +31022,99 @@
         "parameter": "iType"
       }
     ]
+  },
+  {
+    "name": "ADVANCED_FEATURE_FLAGS",
+    "type": "ushort",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "FADF_",
+      "header": "oaidl.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "struct": "SAFEARRAY",
+        "field": "fFeatures"
+      }
+    ]
+  },
+  {
+    "addUsesTo": "tomConstants",
+    "members": [],
+    "uses": [
+      {
+        "interface": "ITextDocument",
+        "method": "Open",
+        "parameter": "Flags"
+      },
+      {
+        "interface": "ITextDocument",
+        "method": "Save",
+        "parameter": "Flags"
+      },
+      {
+        "interface": "ITextRange",
+        "method": "SetPoint",
+        "parameter": "Type"
+      },
+      {
+        "interface": "ITextRange",
+        "method": "GetPoint",
+        "parameter": "Type"
+      },
+      {
+        "interface": "ITextRange",
+        "method": "FindText",
+        "parameter": "Flags"
+      },
+      {
+        "interface": "ITextRange",
+        "method": "FindTextEnd",
+        "parameter": "Flags"
+      },
+      {
+        "interface": "ITextRange",
+        "method": "FindTextStart",
+        "parameter": "Flags"
+      },
+      {
+        "interface": "ITextRange",
+        "method": "ChangeCase",
+        "parameter": "Type"
+      }
+    ]
+  },
+  {
+    "name": "PFN_NUMBERING",
```

Comment:
PARAFORMAT_NUMBERING to match PARAFORMAT_MASK and PARAFORMAT_ALIGNMENT already used in the same struct.